src/SprykerEco/Zed/PunchoutCatalogs/Communication/Form/PunchoutCatalogConnectionFormDataProvider.php

```diff
@@ -53,13 +61,11 @@ public function getData(?PunchoutCatalogConnectionTransfer $punchoutCatalogConne
          */
         return (new PunchoutCatalogConnectionTransfer())
             ->setIsActive(true)
-            ->setType('NOT_IMPLEMENTED')
-            ->setFormat('NOT_IMPLEMENTED')
-            ->setUsername('NOT_IMPLEMENTED');
+            ->setType('NOT_IMPLEMENTED');
     }
 
     /**
-     * @return string[] [idBusinessUnit => label]
+     * @return array [label => idBusinessUnit]
```

Comment:
`int[]`

Reply:
When it's an associative array, and keys are strings IMO we shouldn't use `int[]` typehint because it misleading